clio-client/src/main/scala/org/broadinstitute/clio/client/commands/ClioCommand.scala

```diff
@@ -26,24 +21,37 @@ import shapeless.CNil
 sealed trait ClioCommand
 
 case object GetWgsUbamSchema extends ClioCommand
+case object GetGvcfSchema extends ClioCommand
```

Comment:
I think it would be easier to read if we grouped names defined here by  index, not be operation. E.g. all the WgsUbam names (types/classes/etc) grouped together, followed by all the Gvcf names.

Also I think we want a blank line between each defined name. Or is that not the convention in Scala?